homework6/task.py

```diff
@@ -0,0 +1,36 @@
+phone_book_list = [
+    ["Kate", "+096231453"],
+    ["Jack", "+097312414"],
+    ["Alice", "+063672317"]
+]
+phone_book = dict(phone_book_list)
+stats = len(phone_book)
+while True:
+    print("What do you want to do: ")
+    b = input()
+
+    def add():
+        print("Write name: ")
+        name = input()
+        print("Write his/her number: ")
+        c = input()
+        phone_book[name] = c
```

Comment:
Records shouldn't be overridden according to the task. Please, make additional check here